src/math/FGFunction.cpp

```diff
@@ -74,9 +74,10 @@ class aFunc: public FGFunction
 {
 public:
   aFunc(const func_t& _f, FGFDMExec* fdmex, Element* el,
-        const string& prefix, FGPropertyValue* v)
+        const string& prefix, FGPropertyValue* v, bool constForConstParams=true)
```

Comment:
Remove the `constForCostParams` member, it is not needed (see comment below in `FGFunction::IsConstant`)